cypress/specs/app.spec.ts

```diff
@@ -127,263 +111,250 @@ describe('App', () => {
       cy.findByRole('heading', { name: 'nexus_entry / log_spectrum' }).should(
-        'exist'
+        'be.visible'
       );
-      cy.findByRole('tab', { name: 'NX Spectrum' }).should('exist');
+
       cy.findAllByRole('button', { name: 'Log' }).should('have.length', 2);
 
       if (Cypress.env('TAKE_SNAPSHOTS')) {
-        cy.wait(SNAPSHOT_DELAY);
         cy.matchImageSnapshot('logspectrum');
       }
     });
 
     it('visualize signal and auxiliary signals datasets as NxSpectrum', () => {
-      cy.findByRole('treeitem', { name: /^nexus_entry / }).click();
-      cy.findByRole('treeitem', { name: /spectrum_with_aux/ }).click();
+      cy.selectExplorerNode('nexus_entry');
+      cy.selectExplorerNode('spectrum_with_aux');
 
       cy.findByRole('heading', {
         name: 'nexus_entry / spectrum_with_aux',
-      }).should('exist');
+      }).should('be.visible');
 
       if (Cypress.env('TAKE_SNAPSHOTS')) {
-        cy.wait(SNAPSHOT_DELAY);
         cy.matchImageSnapshot('auxspectrum');
       }
     });
 
     it('visualize dataset with "rgb-image" interpretation as NxRGB', () => {
-      cy.findByRole('treeitem', { name: /^nexus_entry / }).click();
-      cy.findByRole('treeitem', { name: /rgb-image/ }).click();
+      cy.selectExplorerNode('nexus_entry');
+      cy.selectExplorerNode('rgb-image');
 
+      cy.findByRole('tab', { name: 'NX RGB' }).should(...BE_SELECTED);
+      cy.findByRole('figure', { name: 'RGB CMY DGW' }).should('be.visible');
       cy.findByRole('heading', { name: 'nexus_entry / rgb-image' }).should(
-        'exist'
+        'be.visible'
       );
-      cy.findByRole('tab', { name: 'NX RGB' }).should('exist');
-      cy.findByRole('figure', { name: 'RGB CMY DGW' }).should('exist');
 
       if (Cypress.env('TAKE_SNAPSHOTS')) {
-        cy.wait(SNAPSHOT_DELAY);
         cy.matchImageSnapshot('nxrgb');
       }
     });
 
     it('visualize dataset with 1D signal and two 1D axes of same length as NxScatter', () => {
-      cy.findByRole('treeitem', { name: /^nexus_entry / }).click();
-      cy.findByRole('treeitem', { name: /scatter/ }).click();
+      cy.selectExplorerNode('nexus_entry');
+      cy.selectExplorerNode('scatter');
 
+      cy.findByRole('tab', { name: 'NX Scatter' }).should(...BE_SELECTED);
+      cy.findByRole('figure', { name: 'scatter_data' }).should('be.visible');
       cy.findByRole('heading', { name: 'nexus_entry / scatter' }).should(
-        'exist'
+        'be.visible'
       );
-      cy.findByRole('tab', { name: 'NX Scatter' }).should('exist');
-      cy.findByRole('figure', { name: 'scatter_data' }).should('exist');
 
       if (Cypress.env('TAKE_SNAPSHOTS')) {
-        cy.wait(SNAPSHOT_DELAY);
         cy.matchImageSnapshot('nxscatter');
       }
     });
   });
 });
+
+describe('/mock?wide', () => {
+  beforeEach(() => {
+    cy.visit('/mock?wide');
+    cy.waitForStableDOM();
+  });
+
+  it('start with the explorer closed', () => {
```

Comment:
Separating this test in its own `describe` block, as previously, the `visit` inside the test was being displayed as an attempt in the Cypress UI, which was weird.